pkg/kritis/review/review.go

```diff
@@ -72,12 +70,22 @@ func (r Reviewer) ReviewGAP(images []string, gaps []v1beta1.GenericAttestationPo
 			if err != nil {
 				return err
 			}
-			isAuthAttested, _ := r.fetchAndVerifyAttestations(image, auths, pod)
-			if isAuthAttested {
-				imgAttested = true
-				break
+			for _, auth := range auths {
+				transport := AttestorValidatingTransport{Client: r.client, Attestor: auth}
```

Comment:
extract into its own method, for readability?

Reply:
What should I extract?  The loop that goes over auths?  Something else?

Reply:
Yes, the loop that goes over auths please, for readability here and in the method below.

Reply:
Done.